Documentation/user-types.md

```diff
@@ -0,0 +1,59 @@
+## User types
+
+There are many views (aka panes) available through solid-panes, and some do not present a very good UX.
+We wanted to limit some of these views in order to improve the onboarding experience for new users, 
+and to help us with this we've introduced user types.
+These are opt-in and as a new user you aren't assigned any.
+You can self-assign these through Preferences in the Dashboard, 
+and they will be stored in your settings (which are private by default).
+
+The data browser will use these user types to reason which views are applicable for you.
+For instance, if a view lists Developer as audience, 
+the data browser will avoid serving you that view unless you've self-assigned Developer as a user type.
+An exception to this is if the view is the most relevant for a given resource, which makes sharing resources easier.
+
+For now we've introduced the user types "Developer" and "Power User". 
+The former are assigned to views that deal with various data views (e.g., N3, RDF/XML, etc.),
```

Comment:
We decided to link to https://github.com/solid/userguide/#role instead of a local document on this repo. Please make suggestions to changes there instead.